test/imagefsinfo.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+# vim: set syntax=sh:
+
+load helpers
+
+function setup() {
+	setup_test
+	start_crio
+}
+
+function teardown() {
+	cleanup_test
+}
+
+@test "image fs info with default settings should return matching container_filesystem and image_filesystem" {
+	version=$(crictl --version)
+	echo "$version"
+	output=$(crictl imagefsinfo)
+	[ "$output" != "" ]
+
+	container_output=$(jq -e '.status.containerFilesystems[0]' <<< "$output")
+	image_output=$(jq -e '.status.imageFilesystems[0]' <<< "$output")
+	# if these are the same we can directly compare.
+	[ "$container_output" = "$image_output" ]
+}
+
+@test "image fs info with imagestore set should return different filesystems" {
+	version=$(crictl --version)
+	echo "$version"
+
+	stop_crio
+	CONTAINER_IMAGESTORE="$TESTDIR/imagestore" start_crio
```

Comment:
I'd say we remove the `start_crio` from the setup and have no need to stop it again.

Reply:
I need to start it with a different parameter.

Reply:
I mean `stop_crio` would not be required if `setup()` would not container `start_crio`. The other test above can also run `start_crio` while this one calls `CONTAINER_IMAGESTORE="$TESTDIR/imagestore" start_crio`.

Reply:
Ah that is very clear now. Pushed a change for that.